Jenkinsfile

```diff
@@ -120,6 +122,17 @@ pipeline {
             }
           }
         }
+        stage('BDD tests') {
+          when{
+            expression { bdd_test == true }
+          }
+          agent { label 'nixos-mayastor' }
```

Comment:
should this be `'nixos-mayastor-pytest'` ?

Reply:
Renamed to `nixos-mayastor-bdd-tests`.